### PR TITLE
doc: update UP2 sample directory name in create-up2-image.sh

### PR DESCRIPTION
**Pull request by leilu**
Signed-off-by: Lei Lu <leix.lu@intel.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

**Comment by acrnsi** [comment]
OK to verify

File: doc/tutorials/create-up2-images.sh [hunk 1]
```diff
@@ -46,7 +46,7 @@ create_sos_images() {
         return 1
     }
 
-    SOS_BOOTARGS_DEBUG=sos_rootfs/usr/share/acrn/samples/up2/sos_bootargs_debug.txt
+    SOS_BOOTARGS_DEBUG=sos_rootfs/usr/share/acrn/samples/apl-up2/sos_bootargs_debug.txt
```
Comment (dbkinder) [inline doc/tutorials/create-up2-images.sh]: I asked about this in https://github.com/projectacrn/acrn-hypervisor/pull/2667 and was told the sample folder names weren't changing?

Comment (wenlingz) [reply doc/tutorials/create-up2-images.sh]: > I asked about this in #2667 and was told the sample folder names weren't changing?

PR#2761 merged which has modified the up2 name to apl-up2

Comment (dbkinder) [reply doc/tutorials/create-up2-images.sh]: Ha. It looks like the folder name did change later. OK, thanks.

**Comment by wenlingz** [comment]
@dbkinder could you help merge this PR, or we can merge it if the DOC PR which you have approved ?

**Pull request merged**